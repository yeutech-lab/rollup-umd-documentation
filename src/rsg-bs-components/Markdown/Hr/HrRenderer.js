import Hr from 'bootstrap-styled/lib/Hr';
export default Hr;
