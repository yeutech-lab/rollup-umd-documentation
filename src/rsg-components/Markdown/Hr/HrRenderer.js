import React from 'react';
import PropTypes from 'prop-types';

import Styled from 'rsg-components/Styled';

const styles = ({ space, color }) => ({
	hr: {
		borderBottom: [[1, color.border, 'solid']],
		marginTop: 0,
		marginBottom: space[2],
	},
});

export function HrRenderer({ classes }) {
	return <hr className={classes.hr} />;
}
HrRenderer.propTypes = {
	classes: PropTypes.object.isRequired,
};

export default Styled(styles)(HrRenderer);
