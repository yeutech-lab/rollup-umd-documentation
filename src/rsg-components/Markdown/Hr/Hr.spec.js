import React from 'react';

import Hr from './index';

describe('Markdown Hr', () => {
	it('should render a horizontal rule', () => {
		const actual = render(<Hr />);

		expect(actual).toMatchSnapshot();
	});
});
