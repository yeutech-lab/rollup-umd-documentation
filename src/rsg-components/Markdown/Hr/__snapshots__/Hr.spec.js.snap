// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Markdown Hr should render a horizontal rule 1`] = `
<hr
  class="rsg--hr-0"
/>
`;
